molecule/backup_restore/vars/run-backup-vars.yml

```diff
@@ -0,0 +1,17 @@
+tas_single_node_base_hostname: myrhtas
+tas_single_node_cockpit:
+  enabled: false
+tas_single_node_registry_username: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_USERNAME') }}"
+tas_single_node_registry_password: "{{ lookup('env', 'TAS_SINGLE_NODE_REGISTRY_PASSWORD') }}"
+
+tas_single_node_backup_restore:
+  backup:
+    enabled: true
```

Comment:
I believe, that you can have only single `backup-vars.yml` file and switch backup/restore by setting vars in the playbook 
```
- name: Turn backup on
  ansible.builtin.set_fact:
    tas_single_node_backup_restore: "{{ tas_single_node_backup_restore | combine({'backup': {'enabled': true}}, recursive=True) }}"
```